<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pixelDevs</class>
 <widget class="QMainWindow" name="pixelDevs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1392</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pixelDevs Pollen Counter</string>
  </property>
  <property name="toolTipDuration">
   <number>-5</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browseImageButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: none;
	border: 1px solid rgb(85, 255, 255);
	color: rgb(85, 255, 255);
	border-radius: 5px;
	font-family: 'Arial';
	font-weight: bold;
}
QPushButton:hover {
	background-color: none;
	border: 1px solid rgb(63, 190, 190);
	color: rgb(63, 190, 190);
}
</string>
    </property>
    <property name="text">
     <string>BROWSE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileName">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>931</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid rgb(0, 255, 255);
	border-radius: 5px;
	color: rgb(0, 255, 255);
	margin: 0px;
	padding: 10px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>filename.jpg</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(85, 255, 255);
background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>40</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 170, 0);
	border: 1px solid rgb(255, 170, 0);
	color: rgb(0, 0, 0);
	border-radius: 5px;
	font-family: 'Arial';
	font-weight: bold;
}
QPushButton:hover {
	background-color: rgb(190, 127, 0);
	border: 1px solid rgb(190, 127, 0);
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lightPollensCount">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(71, 71, 71);
	color: rgb(255, 255, 255);
	padding: 10px;
	border-radius: 5px;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="darkPollensCount">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(71, 71, 71);
	color: rgb(255, 255, 255);
	padding: 10px;
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lightPollensCountLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>No. of light-colored pollens:</string>
    </property>
   </widget>
   <widget class="QLabel" name="darkPollensCountLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>460</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>No. of dark-colored pollens:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lightColoredImage">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 170, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="darkColoredImage">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>130</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lightPollensCountLabel_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Input Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="conversion">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>540</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(71, 71, 71);
	color: rgb(255, 255, 255);
	padding: 10px;
	border-radius: 5px;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixelMicro">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pixel-to-Micrometer Conversion</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
